### PR TITLE
More tests, more changes to szar.clustering, and a new bin script for the cluster PS (WIP)

### PR DESCRIPTION
**Pull request by dylancromer**
Continuing the work on the cluster calculations. The only significant changes to existing code should be in szar.clustering.

### CODE REVIEW

File: szar/clustering.py [hunk 1]
```diff
@@ -125,6 +126,28 @@ def Norm_Sfunc(self,fsky):
...
+                fine_zs = np.linspace(zgridedges[i], zs[i])
+            else:
+                fine_zs = np.linspace(zgridedges[i], zgridedges[i+1], nsubsamples)
+            ntils = self.ntilde_interpol(fine_zs)
+            dvdz = self.dVdz_fine(fine_zs)
+            integral = simps(dvdz * ntils**2, fine_zs)
+            values[i] = integral * 4 * np.pi * fsky
+        return values
+
+    def v0(self, fsky):
```
Comment (nbatta) [inline szar/clustering.py]: This function should be an integral over dVdz_fine bounded by zbin start to zbin end

File: szar/clustering.py [hunk 2]
```diff
@@ -146,14 +169,38 @@ def ps_bar(self,mu,fsky):
...
-        V0 = self.HMF.dVdz*np.diff(z_arr)*4.*np.pi*fsky #FIX
         nbar = self.ntilde()
+        nbar = np.reshape(nbar, (nbar.size,1))
+
         ps = self.ps_bar(mu,fsky)
-        npfact = np.multiply(ps,nbar)
-        frac = old_div(npfact, (1. + npfact))
-        ans = np.multiply(frac,V0)
+        npfact = np.multiply(nbar, ps)
+        frac = npfact/(1. + npfact)
```
Comment (nbatta) [inline szar/clustering.py]: There's a typo in the paper, this need to be squared

File: szar/clustering.py (hunk 1 above)
Comment (dylancromer) [reply szar/clustering.py]: Weird. I thought when I pushed my commit to my master it would update that. Will fix.

File: szar/clustering.py (hunk 2 above)
Comment (dylancromer) [reply szar/clustering.py]: Weird. I thought when I pushed my commit to my master it would update that. Will fix.

File: szar/clustering.py (hunk 1 above)
Comment (dylancromer) [reply szar/clustering.py]: Fixed.

**Comment by dylancromer** [comment]
I fixed the issue with V_0, and also removed a couple of tests I am done with. I left v_effective.py as I want to expand it into a unit test.

**Pull request merged**